### ISSUE TITLE
Mark as deprecated

### ISSUE THREAD

**Issue by berendt** [ansible-community/ara-web#21]
Since the webinterface in ansible-community/ara is now usable and no progress is visible here I would suggest to deprecate this project. This would help to avoid ambiguity.

**Comment by berendt** [comment]
@dmsimard What do you think?

**Comment by dmsimard** [comment]
Hi @berendt, thanks for the issue and apologies, I've been recovering and been offline for a few weeks.

I wouldn't mark the project as deprecated because although it doesn't have feature parity with the built-in interface, it still (kind of) works.

I've highlighted in different places that the project needs contributors, I think most recently in the [project retrospective](https://ara.recordsansible.org/blog/2019/12/31/2019-in-retrospective-for-ara-records-ansible/) for last year. Until that happens, we could perhaps recommend the built-in interface instead.

Clarifying the built in interface vs ara-web is high on the to-do list for when I come back.

IMO the built-in interface and ara-web should serve different purposes.

In a nutshell, I see the built-in interface as simple, without added dependencies but also limited in scope.

ara-web is a client of the API which happens to be a javascript web application. It's where I would expect features like asynchronous real-time loading of playbook results, fancy graphs (i.e, d3js). It isn't meant to be mutually exclusive with the built-in interface.

I'll flesh out my thoughts when I have more time but hopefully this makes sense.

**Comment by q2dg** [comment]
If I can ask, I would like to be added to ara-web some kind of result filtering (by a part of a written IP address, or by a task name, or sorting by time spent , etc). Thanks a lot.
